gulpfile.js

```diff
@@ -20,7 +20,7 @@ const extensions = ['com.redhat.microprofile.jdt.core', 'com.redhat.microprofile
 const microprofileServerDir = '../quarkus-ls/microprofile.ls/com.redhat.microprofile.ls';
 const extensionDir = '../quarkus-ls/microprofile.jdt';
 
-const quarkusServerExtName = 'com.redhat.quarkus.ls-0.0.1-SNAPSHOT.jar';
+const quarkusServerExtGlob = 'com.redhat.quarkus.ls!(*-sources).jar';
```

Comment:
@fbricon I see that you have spoken about glob, but why can we not generate Quarkus JAR without a version like the current uber JAR?

Reply:
because it's not necessary. we can just drop any jar in there and it'll be picked up, if done properly

Reply:
we could also add the version in the uber jar now, version-less was just more convenient when we had only 1 jar to deal with.